specification/network/resource-manager/Microsoft.Network/stable/2018-02-01/expressRouteCrossConnection.json

```diff
@@ -478,7 +478,7 @@
         "tags": [
           "ExpressRouteCrossConnectionArpTable"
         ],
-        "operationId": "ExpressRouteCircuitsArpTable_List",
+        "operationId": "ExpressRouteCrossConnections_ListArpTable",
         "description": "Gets the currently advertised ARP table associated with the express route circuit in a resource group.",
```

Comment:
We missed this....could you rename express route circuit in the description to express route cross connection?